poly/helper_test.go

```diff
@@ -0,0 +1,106 @@
+package poly
+
+import (
+	"fmt"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/config"
+	"github.com/dedis/crypto/random"
+)
+
+var edward = SUITE
```

Comment:
Unnecessary: just declare a standard test suite like 'suite' or 'testSuite' and refer to that.